backend/src/main/java/io/debezium/configserver/rest/ConnectorResource.java

```diff
@@ -65,13 +68,26 @@ public ConnectorResource() {
     @Path("/connect-clusters")
     @GET
     @Produces(MediaType.APPLICATION_JSON)
+    @APIResponse(
+            responseCode = "200",
+            content = @Content(
+                    mediaType = "application/json",
+                    schema = @Schema(implementation = List.class)
```

Comment:
Is `List` alone that helpful? Can you also specify the element type somehow?

Reply:
I can't say List<String> here, can I? And that's still better than before?

Reply:
Will make a new class
`public class KafkaConnectClusterList extends ArrayList<URI> {`

Reply:
Done

Reply:
WAIT! that doesn't help

Reply:
LOL, ok.

Feel free to rebase and merge yourself, once you figured this one out, LGTM otherwise. Thx!